docs/src/use/configure/migration-guide.md

```diff
@@ -114,6 +113,98 @@ export default [
 ];
 ```
 
+### Processors
+
+In eslintrc files, processors had to be defined in a plugin, and then referenced by name in the configuration. Processors with a name in the special form `.xxx`  were also automatically configured for any files with the extension `xxx`.
+
+In flat config files, processors can still be referenced from plugins by their name, but they can now also be inserted directly into the configuration. Processors will _never_ be automatically configured, and must be explicitly set in the configuration.
+
+As an example with a custom plugin with processors:
+
+```javascript
+// node_modules/eslint-plugin-someplugin/index.js
+module.exports = {
+    processors: {
+        ".md": {
+            preprocess() {},
+            postprocess() {}
+        },
+        "someProcessor": {
+            preprocess() {},
+            postprocess() {}
+        }
+    }
+};
+```
+
+In eslintrc, you would configure as follows:
+
+```javascript
+// .eslintrc.js
+module.exports = {
+    plugins: ["someplugin"],
+    processor: "someplugin/someProcessor"
+};
+```
+
+ESLint would also automatically add the equivalent of the following:
+
+```javascript
+{
+     overrides: [{
+        files: ["**/*.md"],
+        processor: "someplugin/.md"
+     }]
+}
+```
+
+In flat config, the following are all valid ways to express the same:
+
+```javascript
+// eslint.config.js
+import somePlugin from "eslint-plugin-someplugin";
+
+export default [
+    {
+        plugins: { somePlugin },
+        processor: "somePlugin/someProcessor"
+    },
+    {
+        plugins: { somePlugin },
+        // We can embed the processor object in the config directly
+        processor: somePlugin.processors.someProcessor
+    },
+    {
+        // We don't need the plugin to be present in the config to use the processor directly
+        processor: somePlugin.processors.someProcessor
+    }
+];
+```
+
+Note that because the `.md` processor is _not_ automatically added by flat config, you also need to specify an extra configuration element:
+
+```javascript
+{
+    files: ["**/*.md"],
+    processor: somePlugin.processors[".md"]
+}
+```
+
+In flat config, you can even directly embed custom processors without an associated plugin:
+
+```javascript
+// eslint.config.js
+
+export default [
+    {
+        processor: {
+            preprocess() {},
+            postprocess() {}
+        }
+    }
+];
+```
```

Comment:
Let's leave this out. This isn't a tutorial page to show all possible ways to configure flat config, it's a migration guide to go from eslintrc to flat config. Because this isn't a feature that's supported in eslintrc, it shouldn't be mentioned here.

```suggestion
```

Reply:
Sure, it's a new feature in flat config, I wanted to highlight it for users who may have written their own processors locally, and when converting to flat config, can delete the plugin/package that they previously had to wrap the processor in, simplifying that part of their codebase considerably :smile: